packages/linux_sources.rb

```diff
@@ -0,0 +1,24 @@
+require 'package'
+
+class Linux_sources < Package
+  description 'Sources for the Linux kernel'
+  homepage 'https://kernel.org/'
+  version '3.18'
+  source_url 'https://www.kernel.org/pub/linux/kernel/v3.x/linux-3.18.tar.xz'
+  source_sha256 'becc413cc9e6d7f5cc52a3ce66d65c3725bc1d1cc1001f4ce6c32b69eb188cbd'
+
+  def self.install
+    case ARCH
+    when 'x86_64', 'x86'
+      arch = 'x86'
+    when 'armv7l', 'aarch64'
+      arch = 'arm'
+    end
```

Comment:
Why do we define the `arch` variable and never use it?  Unless I'm missing something here?

Reply:
Sorry. Little mistake.

Reply:
Done.